modules/core/src/main/scala/org/tesselation/domain/cluster/programs/TrustPush.scala

```diff
@@ -0,0 +1,34 @@
+package org.tesselation.domain.cluster.programs
+
+import cats.effect.Async
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+
+import org.tesselation.domain.gossip.Gossip
+import org.tesselation.domain.trust.storage.TrustStorage
+import org.tesselation.effects.GenUUID
+
+object TrustPush {
+
+  def make[F[_]: Async: GenUUID](
+    trustStorage: TrustStorage[F],
+    gossip: Gossip[F]
+  ): F[TrustPush[F]] = {
+    val push = new TrustPush[F](trustStorage, gossip) {}
+    Async[F].delay(push)
```

Comment:
It's pure, so it's pointless to wrap it into delay

Reply:
Yep, I think that `make` should return just `TrustPath[F]`.